examples/automm/object_detection/voc_inference.py

```diff
@@ -0,0 +1,28 @@
+import time
+
+from autogluon.multimodal import MultiModalPredictor
+from autogluon.multimodal.utils import from_voc
+
+
+def test_voc_inference(checkpoint_name="faster_rcnn_r50_fpn_2x_coco"):
+    predictor = MultiModalPredictor(
+        hyperparameters={
+            "model.mmdet_image.checkpoint_name": checkpoint_name,
+            "env.num_gpus": 1,
+        },
+        pipeline="object_detection",
+    )
+
+    df = from_voc("VOCdevkit/VOC2007")[:100][["image"]]
+
+    start = time.time()
+    pred = predictor.predict(df, as_pandas=False)  # TODO: disable as_pandas flag for detection
+    print("time usage: %.2f" % (time.time() - start))
+
+    assert len(pred) == len(df)
+    assert len(pred[0]) == 80  # COCO has 80 classes
```

Comment:
Similarly, VOC should have 20 classes.

Reply:
Changed the assert check. Also manually compared the bbox output of class "car" from voc model and coco model on demo image, since there is currently no evaluation support for voc. Voc model's results looks reasonable: there are fewer bounding box predicted comparing with coco, but all high scored bounding boxes are very closed to a bbox in coco's output.